README.md

```diff
@@ -112,6 +112,24 @@ curl -sL https://github.com/binary-install/binstaller/releases/latest/download/i
      sh "$tmpfile" -b "$INSTALL_DIR"; rm -f "$tmpfile")
 ```
 
+#### Test Installation (Dry Run)
+
+Test the installer without actually downloading or installing anything:
```

Comment:
Its' wrong. The installer actually downloads assets with dry run mode. Downloaded assets will be removed though.